### PR TITLE
ZTF Forced Photometry: Flux weighted position

### PR DESCRIPTION
**Pull request by Theodlz**
Adds a dropdown to pick what position to use: the object's, flux weighted alerts only (ignoring FP), and flux weighted ZTF alerts only (ignoring FP).

### CODE REVIEW

File: skyportal/facility_apis/ztf.py [hunk 1]
```diff
@@ -130,19 +130,98 @@ def _build_forced_payload(self, request):
...
+                    and not np.isnan(p.ra)
+                    and not np.isnan(p.dec)
+                    and len(p.streams) > 0
+                    and all(constraint(p) for constraint in additional_constraints)
+                ]
+                if len(photometry) > 0:
+                    # calculate the weighted centroid based on the object's photometry
+                    ras, decs, flux, fluxerr = np.array(photometry).T
+                    snr = flux / fluxerr
+                    ra, dec = np.sum(ras * snr) / np.sum(snr), np.sum(
```
Comment (mcoughlin) [inline skyportal/facility_apis/ztf.py:204]: abs(snr)?

**Review by mcoughlin — APPROVED** [review]
Just one question on SNR.

File: skyportal/facility_apis/ztf.py (hunk 1 above)
Comment (Theodlz) [reply skyportal/facility_apis/ztf.py:204]: Ah yes you are right, plus I said I would use that too, yikes.

Also wondering if I should put a cut on SNR to remove detections of low significance

Comment (mcoughlin) [reply skyportal/facility_apis/ztf.py:204]: Ah yeah abs(snr) > 3 at least.

**Review by mcoughlin — COMMENTED** [review]
Yeah SNR cut good idea.

File: skyportal/facility_apis/ztf.py (hunk 1 above)
Comment (Theodlz) [reply skyportal/facility_apis/ztf.py:219]: added

**Pull request merged**